.gitignore

```diff
@@ -286,20 +287,6 @@ publish/
 # in these scripts will be unencrypted
 PublishScripts/
 
-# NuGet Packages
```

Comment:
This section was hiding some of my new files from git. Removing it didn't expose any NuGet packages?

Reply:
Reverted this and learned about `git add --force`.